Epoch 0:   0%|                           | 0/7 [00:00<?, ?it/s]C:\Users\ADITYA\AppData\Local\Programs\Python\Python313\Lib\site-packages\torch\utils\data\dataloader.py:665: UserWarning: 'pin_memory' argument is set as true but no accelerator is found, then device pinned memory won't be used.
  warnings.warn(warn_msg)
Epoch 0: 100%|███████████████████| 7/7 [01:15<00:00, 10.78s/it]
Traceback (most recent call last):
  File "C:\Users\ADITYA\Desktop\Ml-assignment\train.py", line 315, in <module>
    main()
    ~~~~^^
  File "C:\Users\ADITYA\Desktop\Ml-assignment\train.py", line 263, in main
    val_loss, val_l1, val_ssim, val_perceptual = validate_epoch(model, val_loader, criterion, device)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: not enough values to unpack (expected 4, got 3)
